### DISCUSSION TITLE
大佬这个切换矢量图和影像地图时候 地图都显示白色或者此级别下地区五影像，拉进或者拉远镜头后就好了我试了线上的没问题

### DISCUSSION THREAD

**Discussion by duan6897** [zouyaoji/vue-cesium#502] [Q&A]
`     <vc-layer-imagery name="地图注记"
                        :alpha="alpha"
                        :brightness="brightness"
                        :contrast="contrast"
                        :sort-order="20">
        <vc-imagery-provider-tianditu map-style="cva_c"
                                      token="436ce7e50d27eede2f2929307e6b33c0"></vc-imagery-provider-tianditu>
      </vc-layer-imagery>
      <vc-layer-imagery :name="mapStyle=== 'img_w'?'影像地图':'矢量地图'"
                        :alpha="alpha"
                        :brightness="brightness"
                        :contrast="contrast"
                        :sort-order="10">
        <vc-imagery-provider-tianditu :map-style="mapStyle"
                                      token="436ce7e50d27eede2f2929307e6b33c0"
                                      ref="provider"></vc-imagery-provider-tianditu>`

**Comment by zouyaoji** [comment]
设置一个最大层级就没这问题了。
